unstructured/partition/utils/ocr_models/ocr_interface.py

```diff
@@ -31,26 +31,27 @@ def get_agent(cls) -> OCRAgent:
         The OCR package used by the agent is determined by the `OCR_AGENT` environment variable.
         """
         ocr_agent_cls_qname = cls._get_ocr_agent_cls_qname()
-        try:
-            return cls.get_instance(ocr_agent_cls_qname)
-        except (ImportError, AttributeError):
-            raise ValueError(
-                f"Environment variable OCR_AGENT must be set to an existing OCR agent module,"
-                f" not {ocr_agent_cls_qname}."
-            )
+        return cls.get_instance(ocr_agent_cls_qname)
 
     @staticmethod
     @functools.lru_cache(maxsize=None)
     def get_instance(ocr_agent_module: str) -> "OCRAgent":
         module_name, class_name = ocr_agent_module.rsplit(".", 1)
-        if module_name in OCR_AGENT_MODULES_WHITELIST:
+        if module_name not in OCR_AGENT_MODULES_WHITELIST:
+            raise ValueError(
+                f"Environment variable OCR_AGENT module name {module_name} must be set to a "
+                f"whitelisted module part of {OCR_AGENT_MODULES_WHITELIST}."
```

Comment:
I like the explicit list of modules, that'll make it a lot easier for people to correct.